src/main/webapp/app/shared/reducers/authentication.ts

```diff
@@ -64,7 +70,8 @@ export default (state: AuthenticationState = initialState, action): Authenticati
     case ACTION_TYPES.LOGOUT:
       return {
         ...initialState,
-        showModalLogin: true
+        showModalLogin: true,
+        loggingOut: false
```

Comment:
I think you can also set it to false in the action above (login success) just to be safe

Reply:
ok